packages/filebrowser-extension/schema/browser.json

```diff
@@ -136,6 +136,11 @@
   "title": "File Browser",
   "description": "File Browser settings.",
   "jupyter.lab.shortcuts": [
+    {
+      "command": "filebrowser:go-up",
+      "keys": ["Backspace"],
+      "selector": ".jp-DirListing:focus"
+    },
```

Comment:
Okay, this seems to do the trick: just add another selector to handle the case when the directory is empty and leave the other one alone. I wish I could leave a comment in this JSON file.

This `:focus` selector works for when the directory is empty because the only node (within the directory listing widget) that can receive focus at that point is the directory listing node itself. We cannot use the selector without `:focus` though because then it would get triggered if the user were in the middle of a file name, makes a typo, and then presses backspace to delete the typo. 

I tested this change three ways:

1. Opening a empty folder then pressing backspace
2. Opening a non-empty folder then pressing backspace
3. Renaming a file and pressing backspace while editing the file name